packs/ansible/actions/galaxy.install.yaml

```diff
@@ -0,0 +1,52 @@
+---
+name: "galaxy.install"
+description: "Download & Install role from ansible galaxy"
+runner_type: "local-shell-script"
+entry_point: "ansible_galaxy.py"
+enabled: true
+parameters:
+  kwarg_op:
+    description: "Lock operator type to '--'"
+    type: string
+    default: "--"
+    immutable: true
+  sudo:
+    description: "Lock sudo, the behavior is controlled by ansible 'become_' options"
+    type: boolean
+    default: true
+    immutable: true
+  timeout:
+    description: "Action timeout in seconds. Action will get killed if it doesn't finish in timeout seconds"
+    type: integer
+    default: 300
+  cwd:
+    description: "Working directory where the command will be executed in"
+    type: string
+  action:
+    description: "Action to use"
+    type: string
+    position: 0
+    default: install
+    immutable: true
+  roles:
```

Comment:
Ok, I dirty-fixed it by adding `default` in yaml: 

```yaml
parameters:
  ...
  roles:
    description: "Role(s) to remove (separated by space)"
    type: string
    position: 1
    default: "" #  Added, removes `None` when this positional parameter is not set
```

so `st2 run ansible.galaxy.install` is transformed correctly into `ansible-galaxy install` and the output from ansible is: 

```sh
st2 run ansible.galaxy.install

...
- you must specify a user/role name or a roles file
```